Gemfile

```diff
@@ -7,5 +7,6 @@ group :test do
   gem 'json'
   gem 'minitest'
   gem 'pry-byebug'
+  gem 'toml-rb'
   gem 'yaml'
```

Comment:
Can we remove this gem now?

Reply:
I think it is still need to parse the config as yaml again in the integration test flow